src/distance/constants.js

```diff
@@ -20,6 +23,7 @@ module.exports.SYNONIMS = {
   'дюйм*': 'in',
 
   "'": 'ft',
+  'feet*': 'ft',
```

Comment:
you don't need a star in the end as 'feet' is already a multiple, instead you need a word for the singular 'foot'